### PR TITLE
[ao] quant_type.py fixing public v private

### PR DESCRIPTION
**Pull request by HDCharles**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #88404
* #88400
* #88399
* #88398
* #88397
* #88396
* #88395
* #88394
* #88393
* #88392
* #88391
* #87885
* #87883
* #87521
* #87520
* __->__ #87519

Summary: made _get_quant_type_to_str private

Test Plan: python test/test_public_bindings.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D40709282](https://our.internmc.facebook.com/intern/diff/D40709282)

cc @jerryzh168 @jianyuh @raghuramank100 @jamesr66a @vkuzo @jgong5 @Xia-Weiwen @leslie-fang-intel

### CODE REVIEW

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40709282).

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40709282).

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40709282).

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40709282).

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40709282).

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40709282).

File: torch/ao/quantization/quant_type.py [hunk 1]
```diff
@@ -21,7 +20,7 @@ class QuantType(enum.IntEnum):
 }
 
 # TODO: make this private
```
Comment (jcaip) [inline torch/ao/quantization/quant_type.py:22]: nit: can remove this comment

**Review by jcaip — APPROVED** [review]

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40709282).

**Comment by HDCharles** [comment]
@HDCharles has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40709282).

**Pull request closed without merging**